src/storage/mod.rs

```diff
@@ -1462,7 +1462,7 @@ pub mod test_util {
     ) -> PessimisticLockCommand {
         let primary = keys[0].0.clone().to_raw().unwrap();
         let for_update_ts: TimeStamp = for_update_ts.into();
-        commands::AcquirePessimisticLock::new(
+        commands::acquire_pessimistic_lock::AcquirePessimisticLock::new(
```

Comment:
we should probably re-export all the command types into the commands module rather than referring to them all with the full module path